src/draw/sw/lv_draw_sw_gradient.c

```diff
@@ -137,4 +219,450 @@ void lv_gradient_cleanup(lv_grad_t * grad)
     lv_free(grad);
 }
 
+void lv_gradient_init_stops(lv_grad_dsc_t * grad, const lv_color_t colors[], const uint8_t fracs[],
+                            const lv_opa_t opa[], int num_stops)
```

Comment:
Match the header prototype. I caught this because the render demo broke.
```suggestion
void lv_gradient_init_stops(lv_grad_dsc_t * grad, const lv_color_t colors[], const lv_opa_t opa[],
                            const uint8_t fracs[], int num_stops)
```